SPECIFICATIONS.md

```diff
@@ -755,8 +781,14 @@ message Block {
 }
 ```
 
-Each block contains a `version` field, indicating at which datalog version it
-was generated. Since a Biscuit implementation at version N can receive a valid
+Each `Block` contains the following fields:
+- `symbols`: the list of new [symbols](#symbol-table) introduced in this block.
+- `context`: a free form field with no particular meaning for Biscuit authorization. It can be used to hold application specific data
```

Comment:
```suggestion
- `context`: a free-form field with no particular meaning for Biscuit authorization. It can be used to hold application specific data
```